types/dom-view-transitions/index.d.ts

```diff
@@ -14,3 +14,7 @@ interface ViewTransition {
     updateCallbackDone: Promise<void>;
     skipTransition(): void;
 }
+
+interface CSSStyleDeclaration {
+  viewTransitionName: string
```

Comment:
```suggestion
  viewTransitionName: string;
```

Other than that, lgtm!